directory_header_footer/static/Dit-Pattern-Styling/public/js/script.js

```diff
@@ -725,6 +725,27 @@ dit.components.menu = (new function() {
   var _expanders = [];
   var _accordion = [];
 
+  // Immediately invoked function and declaration.
+  // Because non-JS view is to show all, we might see a brief glimpse of the 
+  // open menu before JS has kicked in to add dropdown functionality. This
+  // will hide the menu when JS is on, and deactivate itself when the JS
+  // enhancement functionality is ready. 
+  dropdownViewInhibitor(true);
```

Comment:
credit to [steven](https://github.com/uktrade/exred/blob/3ea027d9c920617edc0112a58e72dbc35d517958/frontend/components/menu/menu.js#L28)